distro/openshift-template/mt/apicurio-registry-mt-template.yaml

```diff
@@ -780,23 +472,13 @@ parameters:
 
 - name: TENANT_MANAGER_URL
   value: http://tenant-manager:8585
-- name: TENANT_MANAGER_CA_BUNDLE
-  description: path to tenant-manager service CA bundle
-- name: TENANT_MANAGER_CLIENT_SECRET_NAME
```

Comment:
Same with the client secret name, Registry needs these credentials to access TM.

Reply:
restored